src/browser/provider/built-in/chrome/index.js

```diff
@@ -15,12 +17,15 @@ export default {
 
     isMultiBrowser: false,
 
+    _getBrowserName () {
```

Comment:
Firefox and Chrome provides have a lot of code pieces :
```js
openedBrowsers: {},
isMultiBrowser: false,
 async isLocalBrowser () {...}
isHeadlessBrowser (browserId) {...}
 async maximizeWindow (browserId) {...}
```

In this PR you add two more pieces: ` _getBrowserName`  and `isValidBrowserName`.
Is it time to allocate this in a separate base class?